server/@types/licenceApiClientTypes.ts

```diff
@@ -20,7 +20,6 @@ export type Licence =
   | components['schemas']['VariationLicence']
   | components['schemas']['HardStopLicence']
   | components['schemas']['CrdLicence']
-export type LicenceSummaryApproverView = components['schemas']['LicenceSummaryApproverView']
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of. -->
**note:** ‏Listing noted that this is no longer used - double checked and no longer in use so removed.